demos/storybook/stories/toolbar-menu/icon-with-label.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { action } from '@storybook/addon-actions';
+import { GradeA } from '@brightlayer-ui/icons-mui';
+import clsx from 'clsx';
+import createStyles from '@material-ui/core/styles/createStyles';
+import { getDirection } from '@brightlayer-ui/storybook-rtl-addon';
+import makeStyles from '@material-ui/core/styles/makeStyles';
+import { StoryFnReactReturnType } from '@storybook/react/dist/client/preview/types';
+import { text } from '@storybook/addon-knobs';
+import { Theme } from '@material-ui/core/styles/createMuiTheme';
+import { ToolbarMenu } from '@brightlayer-ui/react-components/core/ToolbarMenu';
+import useTheme from '@material-ui/core/styles/useTheme';
+
+const menuItems = [
+    { title: 'Menu Item 1', onClick: action('Item 1 selected') },
+    { title: 'Menu Item 2', onClick: action('Item 2 selected') },
+    { title: 'Menu Item 3', onClick: action('Item 3 selected') },
+];
+
+const menuGroups = [
+    {
+        items: menuItems,
+    },
+];
+
+const useStyles = makeStyles((theme: Theme) =>
+    createStyles({
+        labelContent: {
+            '& > span': {
+                display: 'flex',
+            },
+        },
+        iconMarginRight: {
+            marginRight: `${theme.spacing(1)}px`,
+        },
+        iconMarginLeft: {
+            marginLeft: `${theme.spacing(1)}px`,
+        },
+    })
+);
+export const iconWithLabel = (): StoryFnReactReturnType => {
+    const theme = useTheme();
+    const classes = useStyles(theme);
+    const label = text('label', 'Subtitle');
+    const direction = getDirection();
+    return (
+        <ToolbarMenu
+            classes={{ label: classes.labelContent }}
+            label={
+                <span>
+                    <GradeA className={clsx(direction === 'rtl' ? classes.iconMarginLeft : classes.iconMarginRight)} />
+                    <span>{label}</span>
+                </span>
```

Comment:
If we're going to show a story for 'icon with label', in my mind, that would imply that we have a built-in functionality for including an icon with the label, versus combining the label and icon into the label prop. As an end-user I would be confused how this works if I was just looking at the story and not the source code for it.

Reply:
I will add a new prop `icon` in ToolbarMenu component and update PR.

Reply:
I have added `color` and `icon` prop https://github.com/brightlayer-ui/react-component-library/pull/366/files#diff-0196ae54409f59dbbda8923dc349bc06e8a6e709ade5057a155fa9a7f7a2aa07R76 . Please review.